nltk/tokenize/stanford_segmenter.py

```diff
@@ -82,7 +83,8 @@ def __init__(self,
         self._encoding = encoding
         self.java_options = java_options
         options = {} if options is None else options
-        self._options_cmd = ','.join('{0}={1}'.format(key, json.dumps(val)) for key, val in options.items())
+        self._options_cmd = ','.join('{0}={1}'.format(
```

Comment:
Sure thing. I didn't know how stringent pep8 compliance should be here. I can relax rules a bit.